Jenkinsfile

```diff
@@ -230,10 +229,12 @@ pipeline {
                                         try {
                                             wrap([$class: 'BuildUser']) {
                                                 env.BUILD_USER_ID=env.CHANGE_AUTHOR
-                                                dir(working_dir) {
-                                                    runSctTest(curr_params, builder.region, curr_params.get('functional_tests', false))
-                                                    result = 'SUCCESS'
-                                                    pullRequestSetResult('success', "jenkins/provision_${backend}", 'All test cases are passed')
+                                                timeout(time: 180, unit: 'MINUTES') {
```

Comment:
`3h` is not enough for the k8s functional tests.

Here is how much take took K8S functional tests without SNI ones: `27 passed, 7 skipped in 11103.47s (3:05:03)`

So, probably we should set `5h` here with idea of future tests addition.

Reply:
we probably want the SNI disabled by default, given what was discussed in the weekly call

Reply:
I don't mind to change it for PR checks while we have weekly runs for both configurations.

Reply:
also for the weekly, we might disable it, until they resurrect it.

Reply:
https://github.com/scylladb/scylla-cluster-tests/pull/6903